package.json

```diff
@@ -6,7 +6,7 @@
     "bootstrap": "4.0.0-alpha.6",
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
-    "react-router-dom": "^4.1.2",
+    "react-router-dom": "^5.0.0",
```

Comment:
Be careful when updating dependencies, the reason these come like this in a repo is because they are proven to work together. This will not always be the case so updating your modules when other dependancies might be connected to them or without reading the changes can make for a bad time.